build/DownloadDependencies/CommonProperties.props

```diff
@@ -31,6 +31,13 @@
     <BaseRuleDescFileTargetDirectory>$(MSBuildThisFileDirectory)..\..\src\Rules\Embedded\</BaseRuleDescFileTargetDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SLOOPBinaryFileName>sonarlint-backend-cli-$(EmbeddedSLOOPVersion)-windows_x64.zip</SLOOPBinaryFileName>
+    <SLOOPBinaryDownladDir>$(LOCALAPPDATA)\SLVS_Build_SLOOP</SLOOPBinaryDownladDir>
```

Comment:
Down must be a nice lad since you like him so much

Reply:
I am blind :)